minecraft

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Move minecraft profile if detected at old location
+if [ -d $XDG_DATA_HOME/../.minecraft ]; then
+    if [ ! -f $XDG_DATA_HOME/minecraft/.migrated ]; then
+        echo "Migrating to XDG dirs"
+        mkdir -p $XDG_DATA_HOME/minecraft
+        cp -frl $XDG_DATA_HOME/../.minecraft/* $XDG_DATA_HOME/minecraft/
```

Comment:
Is there any particular reason to do a copy and not a move?

Reply:
Good question. I think there was some, but I don't remember it anymore. Maybe being able to easily rollback to older versions of this Flatpak without breaking compatibility?

Reply:
The reason for a copy was just in-case this got rolled back we didn't have people suddenly lose everything.

We can probably remove this script now since it was only done for migration.